### DISCUSSION TITLE
simGetImages clarification

### DISCUSSION THREAD

**Discussion by emeQee** [microsoft/AirSim#3674] [General]
Hi. I am very new in stereo imaging and AirSim, thus I need some guidance regarding stereo image simulations.

The documentation says, the `simGetImages` call is more complex and "you can get left camera view, right camera view and depth image from left camera in a single API call" (see [Image APIs](https://microsoft.github.io/AirSim/image_apis/#getting-images-with-more-flexibility)). Also the baseline for the stereo camera is fixed at 25 cm (afaik). 

Now my question is:
Can I retrieve left and right images separetly with a single camera? Or do I specifically declare two cameras in `settings.json` and ask the API to get me the images from both cameras?

Background is, that I am trying to simulate a ZED camera with a baseline of 12 cm. I created a `zed-left` camera to get Scene and DepthPerspective images and `zed-right` for Scene only. The camera's are separated by 12 cm as per camera settings. Does the internal 25 cm baseline have any effect on my ZED simulation?

-----

For reference, a partial of my `settings.json`:

``` json
"Cameras": {
    "zed-left": {
        "CaptureSettings": [
            { "ImageType": 0, "Width": 672, "Height": 376, "FOV_Degrees": 90, "MotionBlurAmount": 0 },
            { "ImageType": 2, "Width": 672, "Height": 376, "FOV_Degrees": 90, "MotionBlurAmount": 0 }
        ],
        "X": 0.15, "Y": -0.06, "Z": -1.1,
        "Pitch": -15.0, "Roll": 0, "Yaw": 0
    },
    "zed-right": {
        "CaptureSettings": [
            { "ImageType": 0, "Width": 672, "Height": 376, "FOV_Degrees": 90, "MotionBlurAmount": 0 }
        ],
        "X": 0.15, "Y": 0.06, "Z": -1.1,
        "Pitch": -15.0, "Roll": 0, "Yaw": 0
    }
}
```

**Comment by rajat2004** [comment]
You'll have to use 2 different cameras to fetch the images from, example API call -

```python
responses = client.simGetImages([
    airsim.ImageRequest("zed-left", airsim.ImageType.Scene),
    airsim.ImageRequest("zed-right", airsim.ImageType.Scene)])
```

Settings seems fine, similar to https://github.com/microsoft/AirSim/issues/1790#issuecomment-471779112.
